src/lib/PageControls.js

```diff
@@ -79,13 +99,13 @@ class PageControls extends EventEmitter {
         // show the input box with the current page number selected within it
         this.controlsEl.classList.add(SHOW_PAGE_NUM_INPUT_CLASS);
 
-        this.pageNumInputEl.value = this.currentPageEl.textContent;
+        this.pageNumInputEl.value = this.getCurrentPageNumber();
         this.pageNumInputEl.focus();
         this.pageNumInputEl.select();
 
         // finish input when input is blurred or enter key is pressed
-        this.pageNumInputEl.addEventListener('blur', this.pageNumInputBlurHandler.bind(this));
```

Comment:
@pramodsum bind creates a new function, which caused a new listener to get added each time this was called. It also wasn't being removed correctly as a result.